release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -165,6 +165,18 @@ Previously, secret information was used as the OAuth access token and OAuth auth
 If you are upgrading your cluster to {product-title} 4.6, old tokens from {product-title} 4.5 will still have the secret information exposed in the object name. By default, the expiration for tokens is 24 hours, but this setting can be changed by administrators. Sensitive data can still be exposed until all old tokens have either expired or have been deleted by an administrator.
 ====
 
+
+[id="ocp-4-6-file-integrity-operator"]
+==== File Integrity Operator is now available
+
+The
+xref:../security/file_integrity_operator/file-integrity-operator-understanding.adoc#understanding-file-integrity-operator[File
```

Comment:
This link is dependent upon https://github.com/openshift/openshift-docs/pull/25806, which is still under review